samples/msgext-unfurling-ac-loop-components/nodejs/README.md

```diff
@@ -0,0 +1,140 @@
+  2. Select **New Registration** and on the *register an application page*, set following values:
+      * Set **name** to your app name.
+      * Choose the **supported account types** (any account type will work)
+      * Leave **Redirect URI** empty.
+      * Choose **Register**.
+  3. On the overview page, copy and save the **Application (client) ID, Directory (tenant) ID**. You’ll need those later when updating your Teams application manifest and in the .env.
+  4.  Navigate to the **Certificates & secrets**. In the Client secrets section, click on "+ New client secret". Add a description(Name of the secret) for the secret and select “Never” for Expires. Click "Add". Once the client secret is created, copy its value, it need to be placed in the .env.
+
+### 2. Setup for bot
+In Azure portal, create a [Azure Bot resource](https://docs.microsoft.com/azure/bot-service/bot-service-quickstart-registration).
+    - For bot handle, make up a name.
+    - Select "Use existing app registration" (Create the app registration in Azure Active Directory beforehand.)
+    - __*If you don't have an Azure account*__ create an [Azure free account here](https://azure.microsoft.com/free/)
+    
+   In the new Azure Bot resource in the Portal, 
+    - Ensure that you've [enabled the Teams Channel](https://learn.microsoft.com/azure/bot-service/channel-connect-teams?view=azure-bot-service-4.0)
+    - In Settings/Configuration/Messaging endpoint, enter the current `https` URL you were given by running ngrok. Append with the path `/api/messages`
+
+### 3. Setup
+
+> Note these instructions are for running the sample on your local machine, the tunnelling solution is required because
+the Teams service needs to call into the bot.
+
+1) Run ngrok - point to port 3978
+
+    ```bash
+    ngrok http 3978 --host-header="localhost:3978"
+    ```
+
+### 4. Setup for code
+1) Clone the repository
+
+    ```bash
+    git clone https://github.com/OfficeDev/Microsoft-Teams-Samples.git
+    ```
+
+1) In a terminal, navigate to `samples/msgext-unfurling-ac-loop-components/nodejs`
+
+1) Install modules
+
+    ```bash
+    npm install
+    ```
+
+1) Open .env file from this path folder `samples/msgext-unfurling-ac-loop-components/nodejs` and update 
+   - `{{BOT_ID}}` - Generated from Step 1 (Application (client) ID)is the application app id
+   - `{{BOT_PASSWORD}}` - Generated from Step 1.4, also referred to as Client secret
+
+1) Run your bot at the command line:
+
+    ```bash
+    npm start
+    ```
+
+1) __*This step is specific to Teams.*__
+    - **Edit** the `manifest.json` contained in the  `appPackage` folder to replace your Microsoft App Id (that was created when you registered your bot earlier) *everywhere* you see the place holder string `<<YOUR-MICROSOFT-APP-ID>>` (depending on the scenario the Microsoft App Id may occur multiple times in the `manifest.json`)
```

Comment:
Check if we can rename it to AppManifest

Reply:
Fixed